asset/css/site-nav.css

```diff
@@ -91,7 +92,7 @@
 }
 
 .rotate-icon {
-    color: #0076FF;
+    color: #444444;
```

Comment:
Let's keep `#0076FF` on the `.rotate-icon`.

Reply:
Sure thing! What do you feel about `#444444` vs `#565656`? I am leaning towards the latter.

Reply:
Let's proceed with `#565656`.